libnavigation-core/api/current.txt

```diff
@@ -251,10 +251,8 @@ package com.mapbox.navigation.core.history {
 
   public final class MapboxHistoryReader implements java.util.Iterator<com.mapbox.navigation.core.history.model.HistoryEvent> kotlin.jvm.internal.markers.KMappedMarker {
     ctor public MapboxHistoryReader(String filePath);
-    method public String getFilePath();
```

Comment:
This seems like a breaking change. Should we expose a getter for `filePath` to not break SemVer?

Reply:
yep nice catch! thank you

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6582/files#r1023383239